extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -11,13 +13,266 @@
+>
+> Note that the `attribute` value is a set index `n` that refers to the `_FEATURE_ID_n` vertex attribute semantic. For example, `"attribute": 0` corresponds to `_FEATURE_ID_0`.
+>
+> ![Property Table](figures/feature-table.png)
+>
+> ```jsonc
+> {
+>   "primitives": [
+>     {
+>       "attributes": {
+>         "POSITION": 0,
+>         "_FEATURE_ID_0": 1
+>       },
+>       "indices": 2,
+>       "mode": 4,
+>       "extensions": {
+>         "EXT_mesh_features": {
+>           "featureIds": [ {
+>             "featureCount": 2,
+>             "attribute": 0
+>           } ]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+### Feature ID by Texture Coordinates
+
+*Defined in [mesh.primitive.EXT_mesh_features.schema.json](./schema/mesh.primitive.EXT_mesh_features.schema.json) and [featureIdTexture.schema.json](./schema/featureIdTexture.schema.json).*
+
+Feature ID textures classify the pixels of an image into different features. Some use cases include image segmentation or marking regions on a map. Often per-texel feature IDs provide finer granularity than per-vertex feature IDs, as in the example below.
+
+> **Example:** A building facade, represented by a single quad. The primitive's `baseColorTexture` displays the visible appearance of the building, and its feature ID texture identifies regions of the quad (walls, door, roof, window) as distinct features. Both textures use the same texture coordinates, `TEXCOORD_0`, in this example.
+>
+> <img src="figures/feature-id-texture.png"  alt="Feature ID Texture" width="800">
+>
+> ```jsonc
+> {
+>   "primitives": [
+>     {
+>       "attributes": {
+>         "POSITION": 0,
+>         "TEXCOORD_0": 1
+>       },
+>       "indices": 2,
+>       "material": 0,
+>       "extensions": {
+>         "EXT_mesh_features": {
+>           "featureIds": [ {
+>             "featureCount": 4,     
+>             "texture" : {
+>               "index": 0, 
+>               "texCoord": 0, 
+>               "channels": [0, 1]
+>             }
+>           } ]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+A `featureId` pointing to a feature ID texture extends the glTF [`textureInfo`](../../../../specification/2.0/schema/textureInfo.schema.json) object. The `channels` array contains non-negative integer values corresponding to channels of the source texture that the feature ID consists of. Channels of an `RGBA` texture are numbered 0–3 respectively, although specialized texture formats may allow additional channels. The values from the selected channels are treated as unsigned 8 bit integers, and represent the bytes of the actual feature ID, in little-endian order. 
+
+> **Example:** 
+> If a feature ID defines `"channels": [0, 1]`, then the actual feature ID can be computed as `id = channel[0] | (channel[1] << 8);`. 
+> If a feature ID defines `"channels": [1, 0, 2]`, then the actual feature ID can be computed as `id = channel[1] | (channel[0] << 8) | (channel[2] << 16);`.
```

Comment:
It might be confusing to lead with an example that uses multiple channels to pack the feature ids. Maybe `channels` should be omitted from the first example, and introduced in a second example.